cloudsync/tests/test_cs.py

```diff
@@ -2952,6 +2976,221 @@ def _handle(e: Notification):
             cs.wait(timeout=2)
             assert until()
 
+
+def test_smartsync(scs):
```

Comment:
Can we break this into smaller tests?

Reply:
What's the motivation? the whole test runs in around 1/3 second

Reply:
Readability, this is like 200+ lines. It's hard to follow what's going on or the intended behavior

Reply:
I'll try adding a ton of comments, breaking it up into chunks, that sort of thing, that may be easier and still accomplish your goal